fsw/src/eci_app.c

```diff
@@ -212,454 +215,477 @@ void ECI_APP_MAIN(void);
 **
 **************************************************************************/
 
-/*****************************************************************//**
- * Register Events for this app.
- *
- * \return CFE_SUCCESS if registration was successful.
- ********************************************************************/
-static int32 register_events(void) {
+/*****************************************************************/ /**
+                                                                     * Register Events for this app.
+                                                                     *
+                                                                     * \return CFE_SUCCESS if registration was
+                                                                     *successful.
+                                                                     ********************************************************************/
+static int32 register_events(void)
+{
```

Comment:
Weird comment blocks in this file.

Reply:
The first one was fixed in your last commit, but there's a bunch of them in this file....